target/linux/ipq806x/base-files/etc/board.d/02_network

```diff
@@ -22,6 +22,10 @@ tplink,vr2600v)
 	ucidef_add_switch "switch0" \
 		"1:lan" "2:lan" "3:lan" "4:lan" "6@eth1" "5:wan" "0@eth0"
 	;;
+qxwlan,e5200)
+	ucidef_add_switch "switch0" \
+		"1:lan" "2:lan" "3:lan" "4:lan" "6u@eth1" "5:wan" "0u@eth0"
```

Comment:
Can you give details why this has to be untagged in the commit message?

Reply:
sorry,I don't know what you mean?
Is there a problem with my commit message?
 If there is a problem, it should be that I was negligent in the submission and made a mistake. I am sorry

Reply:
Why 6u and 0u instead of tagged port

Reply:
As Ansuel said, you are using 6u and 0u, so connect these untagged while most other devices are tagged there. So, I assumed you have a reason for that and wanted you to describe that reason in the commit message. If you have no reason for that, please test with the tagged setup (6@eth1 and 0@eth0).

Reply:
@zhangsd20 this still needs to be solved...